org.eclipse.jdt.ls.core/gradle/apt/init.gradle

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+import org.gradle.api.Project
```

Comment:
as a part of init script, the imports like org.gradle.api.* can be just omitted? (https://docs.gradle.org/current/userguide/writing_build_scripts.html#script-default-imports)

Reply:
Yes, they can be removed. Thanks.

Reply:
Looks like the default imports differ between different Gradle versions. When running with 6.2 I got

```
initialization script 'xxx\init.gradle': 41: unable to resolve class GradleVersion 
 @ line 41, column 24.
   			final GradleVersion current = GradleVersion.current().getBaseVersion()
```